packages/convenient_test_manager/integration_test/integration_test.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:convenient_test_common_dart/convenient_test_common_dart.dart';
+import 'package:convenient_test_manager/main.dart';
+import 'package:convenient_test_manager/misc/setup.dart';
+import 'package:convenient_test_manager_dart/services/convenient_test_manager_service.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+var isSetup = false;
+void main() {
+  // color scheme breaks goldens, so we need two variants
+  group('UI GOLDENS', () {
+    myAppGoldenTest(ThemeMode.light);
+    myAppGoldenTest(ThemeMode.dark);
+  });
+}
+
+Future<void> myAppGoldenTest(ThemeMode theme) async {
+  testWidgets('UI golden in theme=$theme', (tester) async {
+    if (!isSetup) {
+      await setup();
+      isSetup = true;
+    }
+
+    await ConvenientTestManagerService().reportInner(ReportCollection(
+      items: [
+        ReportItem(
+          logEntry: LogEntry(testName: 'Test test'),
+        )
+      ],
+    ));
+
+    await tester.pumpWidget(MyApp(theme: theme));
+    await expectLater(find.byType(MyApp),
+        matchesGoldenFile('manager-golden-${theme.name}.png'));
+  });
```

Comment:
Well I guess we have two options here:

1. use `test` (not integration_test). Then, we somehow setup and pump the app *in isolation* for each tests.
2. use flutter_convenient_test to test it. 

In addition, IMHO we need to use the platform brightness override at the start of each testWidgets, so no need to worry about making app rebuild etc

Reply:
Honestly not sure.
For some reason as a test it didn't run on my machine?
Might be just a caching thing, will test more.

Reply:
I see. Because tests are usually much faster than integration tests, so for this specific golden it would be great to use non-integration

Reply:
I see now. But. I just cannot get it to run as a simple test??  I run it `flutter test test/test.dart -v`, it logs:
```
[+501 ms] test 0: connected to test device, now awaiting test result
[        ] test 0: Waiting for test harness or tests to finish
00:04 +0: UI Golden theme=ThemeMode.light                                                                                        
2023-02-19T12:03:31.425095Z|debug|GlobalConfigNullable|parseConfigFile homeDirectory=/home/u
2023-02-19T12:03:31.438401Z|debug|GlobalConfigNullable|parseConfigFile configFilePath=/home/u/config/convenient_test.json
```

and then just seems to wait forever.
Any solutions?